### DISCUSSION TITLE
Stuck keyboard input & Windows 11 taskbar

### DISCUSSION THREAD

**Discussion by Paullusnm** [ppy/osu#25933] [Q&A]
### Type

Other

### Bug description

I have some performance issues and some bugs on my pc. I'm a low-end PC gamer with an Intel UHD Graphics 600 as a GPU and an Intel Celeron N4000 as a CPU. Sometimes when I start a song, my keyboard input is stuck, no keys are pressed, even if I press them. 
Another problem is the taskbar in Windows 11 overlapping osu!lazer, whenever I change the resolution to a lower one.

### Version

2023.1220.0-lazer

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/13722249/compressed-logs.zip)

**Comment by peppy** [comment]
Might help to share a video of what you're seeing.
